Tests/tbDEXTests/HttpClient/tbDEXHttpClientTests.swift

```diff
@@ -83,6 +83,27 @@ final class tbDEXHttpClientTests: XCTestCase {
         XCTAssertNotNil(offerings[0].signature)
     }
     
+    func test_getBalancesWithOneValidBalance() async throws {
```

Comment:
could we add all the failure test cases similar to what you did in https://github.com/TBD54566975/tbdex-js/pull/212 ?